experimental/packages/opentelemetry-exporter-prometheus/package.json

```diff
@@ -59,8 +59,8 @@
     "@opentelemetry/api": "^1.0.3"
```

Comment:
Is it intended that `^1.0.3` is still here but others moved to `^1.1.0`.
I guess via `"@opentelemetry/core": "1.1.1"` it gets implicit api 1.1.0 anyway.